improve vertica content (#2689)
fishtown-analytics/docs.getdbt.com: website/docs/reference/warehouse-setups/vertica-setup.md

title: "Vertica setup"
id: "vertica-setup"
meta:
  maintained_by: Vertica
  authors: Vertica (Former authors:- Matthew Carter, Andy Regan, Andrew Hedengren)
  github_repo: 'https://github.com/vertica/dbt-vertica/'
  pypi_package: 'dbt-vertica'
  min_core_version: 'v1.3.0'
  cloud_support: Not Supported

>   min_supported_version: 'Vertica v12.0'

Reviewer: the intent of this field is to say:
> to use this adapter, you must have at least this version of Vertica.
so this change tells me that you used to be able to use Vertica 10.0, but cannot anymore. Is that the case?
```suggestion
  min_supported_version: 'Vertica 10.0'
```

Author: yes @dataders

>   slack_channel_name: '#adapter-ecosystem'
>   slack_channel_link: 'https://app.slack.com/client/T0VLPD22H/C030A0UF5LM/thread/C030A0UF5LM-1669649490.022339'

Reviewer: this should stay the same. feel free to request a #db-vertica channel in the #channel-requests channel!
```suggestion
  slack_channel_name: 'n/a'
  slack_channel_link: 'https://www.getdbt.com/community/'
```

    <li><strong>dbt Cloud support</strong>: {frontMatter.meta.cloud_support}</li>
    <li><strong>Minimum data platform version</strong>: {frontMatter.meta.min_supported_version}</li>
    </ul>

<h2> Installing {frontMatter.meta.pypi_package} </h2>

pip is the easiest way to install the adapter: <code>pip install {frontMatter.meta.pypi_package}</code>

<p>Installing <code>{frontMatter.meta.pypi_package}</code> will also install <code>dbt-core</code> and any other dependencies.</p>

<h2> Configuring {frontMatter.meta.pypi_package} </h2>

<p>For {frontMatter.meta.pypi_package} specific configuration please refer to <a href={frontMatter.meta.config_page}>{frontMatter.meta.platform_name} Configuration</a> </p>

<p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>

> ### Connecting to Vertica with {frontMatter.meta.pypi_package} 

Reviewer: for inane reasons, you can't use markdown headers when injecting front matter info
```suggestion
<h3> Connecting to {frontMatter.meta.platform_name}   with {frontMatter.meta.pypi_package} </h3>
```
